### ISSUE TITLE
Version conflict with prompt-toolkit 2

### ISSUE THREAD

**Issue by ajslater** [invl/pip-autoremove#19]
```
Traceback (most recent call last):
  File "/usr/local/bin/pip-autoremove", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/pip_autoremove.py", line 106, in main
    autoremove(args, yes=opts.yes)
  File "/usr/local/lib/python2.7/site-packages/pip_autoremove.py", line 21, in autoremove
    dead = list_dead(names)
  File "/usr/local/lib/python2.7/site-packages/pip_autoremove.py", line 29, in list_dead
    graph = get_graph()
  File "/usr/local/lib/python2.7/site-packages/pip_autoremove.py", line 89, in get_graph
    for req in requires(dist):
  File "/usr/local/lib/python2.7/site-packages/pip_autoremove.py", line 95, in requires
    return map(get_distribution, dist.requires())
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 476, in get_distribution
    dist = get_provider(dist)
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 352, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 638, in find
    raise VersionConflict(dist, req)
pkg_resources.VersionConflict: (prompt-toolkit 2.0.4 (/usr/local/lib/python2.7/site-packages), Requirement.parse('prompt-toolkit<1.1.0,>=1.0.10'))
```

if i have prompt-toolkit 2 on my system I can't use pip-autoremove

**Comment by tresni** [comment]
You should see if #16 fixes your issue.

**Comment by tresni** [comment]
Should be addressed by the merge of #42 and the new release, please close if that resolves your issue.

**Comment by ajslater** [comment]
Fixed.
